docs/operate/get-started/supported-hardware/_index.md

```diff
@@ -25,88 +25,27 @@ prev: "/operate/get-started/setup/"
+Any hardware that is not already supported by a Viam module can be added into Viam’s system of modular resources by [creating a new module](/operate/get-started/other-hardware/) that provides a driver for the hardware.
 
 The Viam Registry is the storage and distribution system for not just hardware modules but also software modules (called services), ML models, and ML model training scripts.
 You can browse the [Viam Registry in the Viam app](https://app.viam.com/registry?type=Module).
-{{% /alert %}}
 
-{{% alert title="Support notice" color="note" %}}
-Modules in the list above are officially supported and maintained by Viam if and only if they are marked as "built-in," or if the first part of their model triplet is `viam`.
-{{% /alert %}}
-
-### For use with ESP-32 microcontrollers
-
-The following is a selection of components (some built-ins and some modules) written for use with the Micro-RDK.
-To use any of the built-in components, configure them according to their readmes.
-You can either install the pre-built `viam-micro-server` firmware that ships with a few common modules, or [build your own firmware that combines the Micro-RDK with one or more modules of your choice](/operate/get-started/other-hardware/micro-module/).
-
-<!--prettier-ignore-->
-| Model | Description | Built into the Micro-RDK? | Ships with `viam-micro-server`? |
-| ----- | ----------- | ------------------------------- | --- |
-| `gpio` | A servo controlled by GPIO pins. [Configuration info](/operate/reference/components/servo/gpio-micro-rdk/). | <p class="center-text"><i class="fas fa-check" title="yes"></i></p> | <p class="center-text"><i class="fas fa-check" title="yes"></i></p> |
-| `two_wheeled_base` | A robotic base with differential steering. [Configuration info](/operate/reference/components/base/two_wheeled_base/). Built into the Micro-RDK. | <p class="center-text"><i class="fas fa-check" title="yes"></i></p> | <p class="center-text"><i class="fas fa-check" title="yes"></i></p> |
-| `free_heap_sensor` | Reports the amount of free memory space. [GitHub repo](https://github.com/viamrobotics/micro-rdk/tree/main/examples/modular-drivers/src). | | <p class="center-text"><i class="fas fa-check" title="yes"></i></p> |
-| `wifi_rssi_sensor` | A WiFi signal strength sensor. [GitHub repo](https://github.com/viamrobotics/micro-rdk/tree/main/examples/modular-drivers/src). | | <p class="center-text"><i class="fas fa-check" title="yes"></i></p> |
-| `moisture_sensor` | A moisture sensor. [GitHub repo](https://github.com/viamrobotics/micro-rdk/tree/main/examples/modular-drivers/src). | | |
-| `water_pump` | A water pump. [GitHub repo](https://github.com/viamrobotics/micro-rdk/tree/main/examples/modular-drivers/src). | | |
+Some components are supported by drivers built into `viam-server`, so you won't find them in the registry page.
```

Comment:
Nice way of handling this for now!

Reply:
I was very unsure of how to handle but I wanted to pare down on all the alert boxes haha